content/docs/tooling/cli/_index.md

```diff
@@ -59,6 +61,481 @@ Outputs help text
+
+Without installing and configuring protocol prerequisites and validating data path connectivity
+```yaml
+# Global Driver parameters
+namespace: powerstore
+operator-install: true
+ 
+# Parameters for each PowerStore system
+storage:
+  - endpoint: 10.0.0.1
+    username: user
+  - endpoint: 10.0.0.2
+    username: user
+```
+
+With installing and configuring protocol prerequisites and validating data path connectivity
+```yaml
+# Global Driver parameters
+namespace: powerstore
+operator-install: true
+machineconfig: true
+validate-connectivity: true
+ 
+# Parameters for each PowerStore system
+storage:
+  - endpoint: 10.0.0.1
+    username: user
+  - endpoint: 10.0.0.2
+    username: user
+```
+{{< /collapse >}}
+{{< collapse id="dellctl-install-powerstore-without-machineconfig-validate-output" title="Output the YAML to install CSI Powerstore" card="false" >}}
+Output the YAML to install CSI Powerstore to the console
+```yaml
+# Global Driver parameters
+namespace: powerstore
+operator-install: true
+output: true
+ 
+# Parameters for each PowerStore system
+storage:
+  - endpoint: 10.0.0.1
+    username: user
+  - endpoint: 10.0.0.2
+    username: user
+```
+{{< /collapse >}}
+{{< collapse id="dellctl-install-powerstore-resiliency" title="Install CSI Powerstore with Resiliency" card="false" >}}
+Install CSI Powerstore with CSM Resiliency
+```yaml
+# Global Driver parameters
+namespace: powerstore
+operator-install: true
+modules: resiliency
+ 
+# Parameters for each PowerStore system
+storage:
+  - endpoint: 10.0.0.1
+    username: user
+  - endpoint: 10.0.0.2
+    username: user
+```
+{{< /collapse >}}
+{{< collapse id="dellctl-install-powerstore-observability" title="Install CSI Powerstore with Observability" card="false" >}}
+Install CSI Powerstore with CSM Observability
+```yaml
+# Global Driver parameters
+namespace: powerstore
+operator-install: true
+modules: observability
+ 
+# Parameters for each PowerStore system
+storage:
+  - endpoint: 10.0.0.1
+    username: user
+  - endpoint: 10.0.0.2
+    username: user
+```
+{{< /collapse >}}
+{{< collapse id="dellctl-install-powerstore-authorization" title="Install CSI Powerstore with Authorization" card="false" >}}
+Install CSI Powerstore with CSM Authorization
+```yaml
+# Global Driver parameters
+namespace: powerstore
+operator-install: true
+modules: authorization
+ 
+# Parameters for each PowerStore system
+storage:
+  - endpoint: 10.0.0.1
+    username: user
+  - endpoint: 10.0.0.2
+    username: user
+
+authorization:
+  hostname: csm-authorization.com
+  skipCertificateValidation: false
+  tenantTokenPath: /tmp/token.yaml
+```
+{{< /collapse >}}
+{{< collapse id="dellctl-install-powerstore-replication" title="Install CSI Powerstore with Replication" card="false" >}}
+Install CSI Powerstore with CSM Replication
+```yaml
+# Global Driver parameters
+namespace: powerstore
+operator-install: true
+modules: replciation
+ 
+# Parameters for each PowerStore system
+storage:
```

Comment:
In this replication example, can we add primary and secondary flags to storages ?

Reply:
Fixed.